io.github.ripose.Memento.yml

```diff
@@ -0,0 +1,577 @@
+app-id: io.github.ripose.Memento
+runtime: org.kde.Platform
+runtime-version: '5.15-22.08'
+sdk: org.kde.Sdk
+command: memento
+rename-desktop-file: memento.desktop
+rename-icon: memento
+
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --device=all
+  - --share=network
+  - --socket=pulseaudio
+  - --filesystem=/tmp:rw
+  - --filesystem=xdg-pictures:rw
+  - --filesystem=xdg-videos:rw
+  - --filesystem=home/.var/app/io.github.ripose.Memento/config/memento/scripts:create
+  - --filesystem=host:ro
```

Comment:
why do you need host filesystem permissions?

Reply:
If I don't misunderstand, I need it so I can read local media files outside of xdg-pictures and xdg-videos. io.mpv.Mpv has the permission, so I assume it's useful to my frontend as well.

Reply:
host is a very broad permission.

Reply:
What would you recommend instead? `home` is a nice choice, but it doesn't cover removable or network storage which is a common use-case.